### ISSUE TITLE
[Critical] Text alternative for the informative image is missing

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#630]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87759 |
| **Checkpoint** | 1.1 Text Alternatives › Alternative Text (Informative Images) (1.1.1.b) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 04 Jobs List |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/12/26 at 1:51 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The text alternative for the informative image is missing.
In the Status column, the various checkpoints have a green checkmark indicating success but it is not announced by the screen reader.  It is unclear if when there are an error icon if this will be essential information.

## Recommendation

Rule : 
Images that convey content MUST have programmatically-discernible alternative text.

How To Fix: 
Fix this issue by using ONE of the following techniques:

1. Use aria-label and role="img" to convey the intent, purpose, or meaning of the image.

<p>Have a great day <span class="fonticon smiley" role="img" aria-label="Smiley face"></span></p>

<svg role="img" aria-label="cat">

2. Use CSS clipped text to provide screen reader accessible (non-visible) alternative text.

<a href="https://www.facebook.com/company" class="fb">
<span class="sr-text">Company's Facebook page</span>
</a>

.sr-text {
position: absolute;
clip: rect(1px, 1px, 1px, 1px);
-webkit-clip-path: inset(50%) ;
clip-path: inset(50%);
padding: 0;
border: 0;
height: 1px;
width: 1px;
white-space: nowrap;
overflow: hidden;
}

Reference: 
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/images/

Background: 
People who are blind cannot see images on a page. In order to give people who cannot see an image access to the information conveyed by the image, it must have a text alternative. The text alternative must describe the information or function represented by the image. Screen readers can then use the alternative text to convey that information to the screen reader user.

For informative images - images that convey information - the alternative text must communicate the intent, purpose, or meaning of the image in a way that serves as a true alternative for the image.

It is not possible to provide alternative text by adding an alt attribute directly to CSS background images. Images that convey content should not generally be placed in CSS. However, if it is necessary to do so, you can add alternative text to informative and actionable CSS images using other methods.

## Screenshots

![04_informative](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87759/04_informative.png)
